Gems/AWSCore/Code/Platform/Common/GetCertsPath_Null.cpp

```diff
@@ -25,4 +25,4 @@ namespace AWSCore
             return ""; // no-op
         }
     } // namespace Platform
-} // namespace GridMate
+} // namespace AWSCore
```

Comment:
is this the right namespace?

Reply:
Yep, this just corrects what I suspect was a copy paste error.